jars/sources/ExerelinCore/exerelin/campaign/intel/raid/NexRaidIntel.java

```diff
@@ -329,7 +329,16 @@ public boolean shouldRetreatIfOvermatched() {
 		
 		return true;
 	}
-	
+
+	@Override
+	protected void notifyEnding() {
+		sendOutcomeUpdate();
+		if (ExerelinModPlugin.isNexDev) {
+			Global.getSector().getCampaignUI().addMessage("notifyEnding() called in NexRaidIntel");
+		}
+		super.notifyEnding();
+	}
+
```

Comment:
I'm not sure if anything except for sendOutcomeUpdate() is necessary, but I don't fully understand how these methods work, so I added super.notifyEnding() just in case. It seems to work.

Reply:
I also don't seem to understand how to properly use github comments...